src/libs/context.js

```diff
@@ -1,4 +1,3 @@
-import nanoid from 'nanoid/non-secure'
```

Comment:
My linter suggested this was not being used in this file anymore so I removed it.